tools/ide-catalog-generator/pom.xml

```diff
@@ -17,11 +17,15 @@
 		<maven>2.0.9</maven>
 	</prerequisites>
 
+	<properties>
+		<camel.version>2.18.1.fuse-000010</camel.version>
```

Comment:
qualifier has changed, several places need to be changed accordingly I think, searching for ".redhat is providing 7 results, FuseIntegrationProjectWizardRuntimeAndCamelPage.isCompatible seems to be th cornerstone of that